draft-irtf-cfrg-aegis-aead.md

```diff
@@ -456,7 +456,7 @@ The `Decrypt` function decrypts a ciphertext, verifies that the authentication t
 
 Security:
 
-- If tag verification fails, the decrypted message and wrong message authentication tag MUST NOT be given as output. The decrypted message MUST be overwritten with zeros.
+- If tag verification fails, the decrypted message and wrong message authentication tag MUST NOT be given as output. The decrypted message MUST be overwritten with zeros before being returned.
```

Comment:
As said in the issue, should we add `if this is possible in your programming language` to the end?

Reply:
If this is not possible, it's probably due to implementing this with an insecure API (pushing the result into a queue before verifying the tag) rather than a specific language. "if this is possible" would also contradict MUST NOT. So, I'd leave it as is.